packages/salesforcedx-utils-vscode/src/helpers/traceFlagsRemover.ts

```diff
@@ -50,4 +50,8 @@ export class TraceFlagsRemover {
       }
     }
   }
+
+  public async removeTraceFlag(traceFlagId: string) {
```

Comment:
I don't think there's much value of having this as a method on the class since it's not interacting with the rest of the class's stuff.

export const removeTraceFlag (conn: Connection, traceFlagId:string) and then use that.  then you don't have to go through the whole getInstance() just to do the simple thing.

while we're in here, I see `removeNewTraceFlags` is unused.

Reply:
actually, a lot from this looks unused.